client/app/widgets/Resources/index.jsx

```diff
@@ -3,25 +3,84 @@ import React from 'react';
 import css from './Resources.scss';
 import { Resource } from '../../components/Resource';
 import { Utils } from '../../utils';
+import { InputTag } from '../../components/Input/InputTag';
 
 export type Props = {
   resources: any,
 };
 
-export const Resources = (props: Props) => {
-  const { resources } = props;
-  return (
-    <div className={css.gridThree}>
-      {resources.map(resource => (
-        <div className={css.gridThreeItem} key={Utils.randomString()}>
-          <Resource
-            tagged
-            tags={resource.languages.concat(resource.tags)}
-            title={resource.name}
-            link={resource.link}
+export type State = {
+  checkboxes: any
+};
+
+export type Selected = {
+  checked: boolean, 
+  id: string, 
+  key: string, 
+  label: string, 
+  value: string
+}
+
+export class Resources extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { checkboxes: this.createCheckboxes() };
+  }
+  
+  createCheckboxes = () => {
+    const { resources } = this.props;
+    const tagsList = [...new Set(resources.map(res => res.tags.concat(res.languages)).reduce((acc, val) => acc.concat(val), []))];
```

Comment:
ESLint probably complained that this line is too long. Let's fix this. Same for the next line.